### PR TITLE
common/command testing v1

### PR DESCRIPTION
**Pull request by timvaillancourt**

### CODE REVIEW

**Comment by timvaillancourt** [comment]
Travis CI failure
```
=== RUN   TestNewCommand
--- FAIL: TestNewCommand (0.00s)
	assertions.go:254: 
			Error Trace:	command_test.go:27
			Error:      	Received unexpected error:
			            	user: unknown user Travis CI User
			Test:       	TestNewCommand
			Messages:   	.New() should not return an error
```

File: common/util.go [hunk 1]
```diff
@@ -47,3 +50,21 @@ func DoStop(stop *chan bool) bool {
 		return false
 	}
 }
+
+// GetUserId returns the numeric ID of a system user
+func GetUserId(userName string) (int, error) {
```
Comment (golangci) [inline common/util.go]: func GetUserId should be GetUserID

File: common/util.go [hunk 2]
```diff
@@ -47,3 +50,21 @@ func DoStop(stop *chan bool) bool {
...
+	u, err := user.Lookup(userName)
+	if err != nil {
+		return -1, err
+	}
+	return strconv.Atoi(u.Uid)
+}
+
+// GetGroupID returns the numeric ID of a system group
```
Comment (golangci) [inline common/util.go]: comment on exported function GetGroupId should be of the form "GetGroupId ..."

File: common/util.go [hunk 3]
```diff
@@ -47,3 +50,21 @@ func DoStop(stop *chan bool) bool {
...
+func GetGroupId(groupName string) (int, error) {
```
Comment (golangci) [inline common/util.go]: func GetGroupId should be GetGroupID

File: common/util.go (hunk 1 above)
Comment (golangci) [inline common/util.go]: func GetUserId should be GetUserID

File: common/util.go (hunk 2 above)
Comment (golangci) [inline common/util.go]: comment on exported function GetGroupId should be of the form "GetGroupId ..."

File: common/util.go (hunk 3 above)
Comment (golangci) [inline common/util.go]: func GetGroupId should be GetGroupID

File: common/util.go (hunk 1 above)
Comment (golangci) [inline common/util.go]: func GetUserId should be GetUserID

File: common/util.go (hunk 2 above)
Comment (golangci) [inline common/util.go]: comment on exported function GetGroupId should be of the form "GetGroupId ..."

File: common/util.go (hunk 3 above)
Comment (golangci) [inline common/util.go]: func GetGroupId should be GetGroupID

**Pull request merged**